src/script/interpreter.cpp

```diff
@@ -591,7 +652,42 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
                     break;
                 }
 
-                case OP_NOP1: case OP_NOP4: case OP_NOP5:
```

Comment:
`case OP_NOP1:` was removed (by accident?). Should it be added back to L691?

Reply:
Git diff. Making life unnecessarily hard sometimes.